pages/admin/dashboard/modules.tsx

```diff
@@ -0,0 +1,92 @@
+import React from "react";
+import {
+    Flex,
+    Text,
+    Button,
+    Spacer,
+    SimpleGrid,
+    Stack,
+} from "@chakra-ui/react";
+
+import { ModuleCard } from "@components/ModuleCard";
+import { Page } from "types/Page";
+import { AdminLayout } from "@components";
+
+const Modules: Page = () => {
+    const modules = React.useMemo(
+        () => [
```

Comment:
Can we move this data to a separate file? Right now all our static data is in `data/`